src/python/pants/base/exception_sink.py

```diff
@@ -237,16 +256,24 @@ def _iso_timestamp_for_now(cls):
   # NB: This includes a trailing newline, but no leading newline.
   _EXCEPTION_LOG_FORMAT = """\
 timestamp: {timestamp}
-args: {args}
+process title: {process_title}
+sys.argv: {args}
+bootstrap options: {bootstrap_options}
 pid: {pid}
 {message}
 """
 
   @classmethod
   def _format_exception_message(cls, msg, pid):
+    if cls._bootstrap_option_values:
+      bootstrap_fmt = cls._bootstrap_option_values._debug_dump()
```

Comment:
This is going to be a bit too noisy probably?

Reply:
It totally is, and this is why the testing has the `nonce` thing at all. I think the first suggestion you described (to capture the relevant fields) is the one to go for here, that obviates the `_debug_dump()` method.

Reply:
Removed!